doc/source/dev/contributor/quickstart_pip.rst

```diff
@@ -116,12 +116,15 @@ you cloned the source code into. Download the submodules::
 
     git submodule update --init
 
-And build SciPy (this takes a while)::
+Now configure the build system (note that BLAS/LAPACK arguments are needed
+because we used system-provided libraries; if you use e.g. OpenBLAS, these
+arguments can be omitted)::
 
-    python setup.py build
+    meson setup build -Dblas=blas -Dlapack=lapack
```

Comment:
This seems unnecessarily confusing for newcomers. Why not change the instructions higher up to use OpenBLAS instead? That is faster and better tested by now, because it's the default everywhere.

Then you can just get rid of this whole bit, it shouldn't be in quickstart.

Reply:
Okay, done in [a54595c](https://github.com/scipy/scipy/pull/15569/commits/a54595c4a6a63dca2792eb27edd2cc92da3b9c50). Using the system provided openblas smokes out what looks like a too-tight tolerance test, will send a separate PR.

FTR, there is value in keeping netlib/ATLAS in support. If nothing else, it provides a stable baseline. Not actively developed => no new bugs, no need to worry about versions etc. Upgrading OpenBLAS is not trivial (at least outside of the conda world); heck, even `sudo apt-cache` lists a dozen alternatives, which one is needed? Anyway, this is a separate discussion, so let's not mix it into the quickstart guide :-).

Reply:
Yes I agree that we need to keep supporting all BLAS/LAPACKs we can. And we need a separate doc page on that - just not in quickstart.

Also, I think the recent support for Netlib BLAS/LAPACK shows we may want to add one CI job using them. I'll add it to the CI matrix. Same for MKL perhaps. We need a larger CI matrix to cover more flavors of stuff without increasing the number of jobs.